explos/channel-manager/teritoriAPI.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+)
+
+type RefreshData struct {
```

Comment:
not very important, but when I develop `main` packages (not importable), I have the habit of unexport the structs, the methods etc.

In general, it's a good thing to just export what's needed. And a good practice to start unexported, and export when we have an valid reason.